src/main/java/cms/gov/madie/measure/resources/BundleController.java

```diff
@@ -0,0 +1,43 @@
+package cms.gov.madie.measure.resources;
+
+import cms.gov.madie.measure.exceptions.ResourceNotFoundException;
+import cms.gov.madie.measure.repositories.MeasureRepository;
+import cms.gov.madie.measure.services.BundleService;
+import cms.gov.madie.measure.utils.ControllerUtil;
+import gov.cms.madie.models.measure.Measure;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.security.Principal;
+import java.util.Optional;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+public class BundleController {
+
+  private final MeasureRepository measureRepository;
+
+  private final BundleService bundleService;
+
+  @GetMapping(path = "/measures/{measureId}/bundles", produces = MediaType.APPLICATION_JSON_VALUE)
+  public ResponseEntity<String> getMeasureBundle(
+      @PathVariable String measureId,
+      Principal principal,
+      @RequestHeader("Authorization") String accessToken) {
+    Optional<Measure> measureOptional = measureRepository.findById(measureId);
```

Comment:
Add an audit log entry, please.

Reply:
Adding a new log statement, Everything related to bundles in this PR is just refactoring.